packages/grafana-ui/src/components/PanelChrome/PanelChrome.story.tsx

```diff
@@ -51,7 +51,7 @@ function renderPanel(name: string, overrides?: Partial<PanelChromeProps>) {
 
   return (
     <PanelChrome {...props}>
-      {(innerWidth, innerHeight) => {
+      {(innerWidth: number, innerHeight: number) => {
```

Comment:
it's not a major problem cause it's kind of a niche pattern, but do we know why these don't get inferred correctly anymore? 🤔

Reply:
I'm pretty sure this is specific to storybook/this story, with how `props` is spread into the component. Typescript cannot narrow the props down to know whether it should be ReactNode or a render callback.

If you explicitly declare width and height props, it types the render callback correctly.

```ts
<PanelChrome width={1} height={2}>
  {(innerWidth, innerHeight) => {
    return <div />;
  }}
</PanelChrome>
```